src/views/Setup.vue

```diff
@@ -58,8 +58,8 @@ export default {
 				.catch((error) => {
 					logger.error('Could not create account', { error })
 
-					if (error.message) {
-						this.error = error.message
+					if (error.data.message) {
```

Comment:
```suggestion
					if (error?.data.message) {
```

just in case a non-axios error is thrown, those don't have a `data` prop